wfe/wfe.go

```diff
@@ -1116,12 +1126,12 @@ func (wfe *WebFrontEndImpl) makeAuthorizations(order *core.Order, request *http.
 	// Lock the order for reading
 	order.RLock()
 	// Create one authz for each name in the order's parsed CSR
-	for _, name := range order.Names {
+	for _, name := range order.Identifiers {
```

Comment:
I think before this step Pebble also needs to ensure there aren't any duplicate IP address identifiers in the order.